ARM-wvd-templates/Update exisiting WVD host pool/AddVirtualMachinesTemplate.json

```diff
@@ -0,0 +1,537 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "nestedTemplatesLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/test_wvd_springUpdate_template/spring20-wvd-templates/nestedtemplates/"

```

Comment:
Needs to be updated to https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/nestedtemplates/ per other comment.

Reply:
ok